tools/psa/build_tfm.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/python
+"""
+Copyright (c) 2019 ARM Limited. All rights reserved.
+
+SPDX-License-Identifier: Apache-2.0
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import os
+from os.path import join, abspath, dirname, isdir
+import sys
+import subprocess
+import logging
+
+logger = logging.getLogger('TF-M-Builder')
+logging.basicConfig(level=logging.INFO,
+                    format='[%(name)s] %(asctime)s: %(message)s.',
+                    datefmt='%H:%M:%S')
+
+ROOT = abspath(join(dirname(__file__), os.pardir, os.pardir))
+sys.path.insert(0, ROOT)
+
+TF_M_BUILD_DIR = join(ROOT, os.pardir, 'tfm_build_dir')
+VERSION_FILE_PATH = join(ROOT, 'features/FEATURE_PSA/FEATURE_TFM')
```

Comment:
The commit message needs updating. It still says "components".

Reply:
@Patater fixed.